Libplanet.Tools/Mpt.cs

```diff
@@ -23,6 +23,14 @@ internal enum SchemeType
 
     public class Mpt
     {
+        private const string KeyValueStoreURIExample =
+            "(e.g., <kv-store-type>://<kv-store-path>, rocksdb:///path/to/kv-store)";
+
+        private const string KeyValueStoreArgumentDescription =
+            "The alias registered through `mpt add` command or the URI included " +
```

Comment:
- To make it clearer, it'd better show the top-level command `planet`.
- For [typographic consistency][1], I recommend you to use backquotes for opening quotes, and single quotes for closing quotes (FYI [it's quite long-historied tradition on ASCII typography][2]).

```suggestion
            "The alias name registered through `planet mpt add' command or the URI included " +
```

[1]: https://github.com/planetarium/libplanet/blob/main/Libplanet.Tools/Apv.cs#L29-L45
[2]: https://en.wikipedia.org/wiki/Grave_accent#As_surrogate_of_apostrophe_or_(opening)_single_quote